sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/DateTimeUtils.scala

```diff
@@ -19,8 +19,11 @@ package org.apache.spark.sql.catalyst.util
 
 import java.sql.{Date, Timestamp}
 import java.text.{DateFormat, SimpleDateFormat}
+import java.time._
```

Comment:
If there aren't like 10 classes to import, I'd make them explicit. At least, the following line is currently redundant.

Reply:
yah, IntelliJ IDEA collapses the imports automatically. I will try to revert them back.